enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -897,6 +897,10 @@ func (p *PebbleCache) Metadata(ctx context.Context, r *resource.ResourceName) (*
 		return nil, err
 	}
 
+	if md.GetStoredSizeBytes() != md.GetFileRecord().GetDigest().GetSizeBytes() {
+		log.Debugf("Pebble metadata size mismatch: %v", md)
```

Comment:
We don't enable Debug level logs in prod so let's make this an Info level log.

BTW, try to use the log.Ctx* functions if there's a context already available. It makes it possible for us to enrich our logs with more metadata.